<a href="https://colab.research.google.com/github/revati2013/Assignment03_hypothesis_testing/blob/main/Hypothesis_testing_3.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Q3. Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.


In [1]:
import pandas as pd
import numpy as np
from scipy.stats import chi2_contingency


In [2]:
data=pd.read_csv('BuyerRatio.csv')

In [3]:
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [4]:
data1=data.drop(['Observed Values'],axis=1)

In [5]:
data1

East  West  North  South
0    50   142    131     70
1   435  1523   1356    750

In [8]:
obs=np.array(data1)
obs

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

In [7]:
chi2_contingency(obs) 

(1.595945538661058,
 0.6603094907091882,
 3,
 array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
        [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))

Inference: As (p-value = 0.6603) > (α = 0.05); Accept the Null Hypothesis i.e. Independence of categorical variables Thus, male-female buyer rations are similar across regions and are not related